### PR TITLE
Issue #DP-784 fix: Telemetry changes and html changes based on new design 

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/mvc-filter/mvc-filter.component.html [hunk 1]
```diff
@@ -3,11 +3,10 @@
     <div class="d-flex flex-w-wrap">
       <div class="fs-0-785 cursor-pointer pr-20 font-weight-bold" (click)="showfilter()"
         appTelemetryInteract
-        [telemetryInteractEdata]="programTelemetryService.getTelemetryInteractEdata('filter-accordion', 'click', 'mvc-explore')"
-        [telemetryInteractObject]="programTelemetryService.getTelemetryInteractObject(sessionContext.collectionUnitId, 'Content', '1.0', {l1: sessionContext.collection})"
+        [telemetryInteractEdata]="programTelemetryService.getTelemetryInteractEdata('filter-accordion', 'click', 'library',{'isFilterShow':!isFilterShow})"
+        [telemetryInteractObject]="programTelemetryService.getTelemetryInteractObject(undefined, 'Content', '1.0', {l1:sessionContext.collectionUnitId,l2: sessionContext.collection})"
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/cbse-program/components/mvc-filter/mvc-filter.component.html:7]: @vaibhavbhuva why are you sending `undefined` ?

File: src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.html [hunk 2]
```diff
@@ -71,8 +71,8 @@ <h5 class="font-weight-bold">{{resourceService.frmelmnts.lbl.sorryThereIsNoMatch
       <h6 class="sb-color-gray-300 font-weight-normal mt-8">{{resourceService.frmelmnts.lbl.changingFilterHelpsYouFindMoreContent}}</h6>
       <button class="sb-btn sb-btn-secondary sb-btn-normal mt-16" (click)="openFilter();" 
         appTelemetryInteract
-        [telemetryInteractEdata]="programTelemetryService.getTelemetryInteractEdata('change-filter', 'click', 'mvc-explore')"
-        [telemetryInteractObject]="programTelemetryService.getTelemetryInteractObject(collectionUnitId, 'Content', '1.0', {l1: collectionId})"
+        [telemetryInteractEdata]="programTelemetryService.getTelemetryInteractEdata('change-filter', 'click', 'library')"
+        [telemetryInteractObject]="programTelemetryService.getTelemetryInteractObject(undefined, 'Content', '1.0', {l1:collectionUnitId,l2: collectionId})"
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.html:75]: @vaibhavbhuva same here?

File: src/app/client/src/app/modules/cbse-program/components/mvc-filter/mvc-filter.component.html (hunk 1 above)
Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/cbse-program/components/mvc-filter/mvc-filter.component.html:7]: `getTelemetryInteractObject` ---> passing as **undefined** bcz every time object Id will be not available and this function also handling falsy values that I have tested and verified the same.

File: src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.html (hunk 2 above)
Comment (vaibhavbhuva) [reply src/app/client/src/app/modules/cbse-program/components/mvc-library/mvc-library.component.html:75]: @pallakartheekreddy Updated here: https://github.com/Sunbird-Ed/creation-portal/pull/749#discussion_r458532234

**Pull request merged**